File: LeetCode/2114_Maximum_Number_of_Words_Found_in_Sentences/solution.js
/**
 * @param {string[]} sentences
 * @return {number}
 */
var mostWordsFound = function (s) {

    let max=0;
    for (let i = 0; i < s.length; i++) {
        let temp = 0;
        for (let j = 0; j < s[i].length; j++) {
            if( (j==0 && s[i][j]!=" ") || (s[i][j]!=" " && s[i][j-1]==" ") ) temp ++;
        console.log(temp)
        }
        max=max>temp?max:temp;
    }
    return max;
};

File: LeetCode/2114_Maximum_Number_of_Words_Found_in_Sentences/solution.cpp
class Solution {
public:
    int mostWordsFound(vector<string>& sentences) {
        int max = 0;
        for (int i = 0; i < sentences.size(); i++) {
            int temp = 0;
            for (int j = 0; j < sentences[i].size(); j++) {
                if(sentences[i][j] == ' ') temp++;
            }
            max = max > temp ? max : temp;
        }
        return max+1;
    }
};